azurerm/internal/services/media/media_live_output_resource_test.go

```diff
@@ -0,0 +1,185 @@
+package media_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/media/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type LiveOutputResource struct{}
+
+func TestAccLiveOutput_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_media_live_output", "test")
+	r := LiveOutputResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeAggregateTestCheckFunc(
+				check.That(data.ResourceName).Key("name").HasValue("Output-1"),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccLiveOutput_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_media_live_output", "test")
+	r := LiveOutputResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeAggregateTestCheckFunc(
+				check.That(data.ResourceName).Key("name").HasValue("Output-1"),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func TestAccLiveOutput_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_media_live_output", "test")
+	r := LiveOutputResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.complete(data),
+			Check: resource.ComposeAggregateTestCheckFunc(
+				check.That(data.ResourceName).Key("manifest_name").HasValue("testmanifest"),
+				check.That(data.ResourceName).Key("hls_fragments_per_ts_segment").HasValue("5"),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
```

Comment:
could we add a update test that goes basic -> complete -> basic configs

Reply:
OK